experimental/packages/exporter-trace-otlp-http/test/browser/CollectorTraceExporter.test.ts

```diff
@@ -33,6 +33,8 @@ import {
   OTLPExporterError,
 } from '@opentelemetry/otlp-exporter-base';
 import { IExportTraceServiceRequest } from '@opentelemetry/otlp-transformer';
+import { FetchMockStatic, MockCall } from 'fetch-mock/esm/client';
+const fetchMock = require('fetch-mock/esm/client').default as FetchMockStatic;
```

Comment:
I'm wondering: what is the reason to use `fetch-mock/esm/client` instead of `fetch-mock` :thinking:

Reply:
I'm not familiar with it and don't know about a better way in TypeScript + karma browser test.
If you have any suggestions,  I hope to update the code.

Reply:
Generally, unless the package states that it supports "named" (entry-points) using `fetch-mock/esm/client` will either not work or may be broken in a future release of their package (because they move the code internally)

So if they are "not" exported via the main entry point (index.ts normally), then it's likely that these should be considered to be "internal" classes and we may get broken if/when they refactor the code that uses / defines them.

Reply:
Yes. Your note is correct in general.

However, loading fetch-mock in this way is described in the official document;

http://www.wheresrhys.co.uk/fetch-mock/#importing-the-right-file

> If your client-side code or tests do not use a loader that respects the `browser` field of `package.json` use `require('fetch-mock/es5/client')` or `import fetchMock from 'fetch-mock/esm/client'`.

I guess that the author will keep the filename in future releases.

On the other hand, if you know, would you please tell me the way to update loader settings to avoid loading internal files directly?